test/opf/sdpwrm.jl

```diff
@@ -65,9 +65,13 @@ Test dual feasibility of SDPWRM problem.
 
 This test is executed on the 5 bus system.
 """
-function _test_sdpwrm_DualFeasibility()
+function _test_sdpwrm_DualFeasibility(OPF::Union{Type{PGLearn.SDPOPF}, Type{PGLearn.SparseSDPOPF}})
     T = Float128
-    data = PGLearn.OPFData(make_basic_network(pglib("5_pjm")))
+    if OPF == PGLearn.SDPOPF
+        data = PGLearn.OPFData(make_basic_network(pglib("5_pjm")))
+    else
+        data = PGLearn.OPFData(make_basic_network(pglib("14_ieee")))
+    end
```

Comment:
Why not run on the same system all the time?

Reply:
Since we've been using 5_pjm for `SDPOPF`, I'm changing it to use 5_pjm for `SparseSDPOPF` too.